ktouch 18.12.0 not building

## Issue description

Cannot install ktouch.

```
CMake Error at /nix/store/yjj7f7lypknwzbxvkvjkf8aa0av6ss0d-qtbase-5.12.0-dev/lib/cmake/Qt5/Qt5Config.cmake:28 (find_package):
  By not providing "FindQt5XmlPatterns.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "Qt5XmlPatterns", but CMake did not find one.

  Could not find a package configuration file provided by "Qt5XmlPatterns"
  with any of the following names:

    Qt5XmlPatternsConfig.cmake
    qt5xmlpatterns-config.cmake

  Add the installation prefix of "Qt5XmlPatterns" to CMAKE_PREFIX_PATH or set
  "Qt5XmlPatterns_DIR" to a directory containing one of the above files.  If
  "Qt5XmlPatterns" provides a separate development package or SDK, be sure it
  has been installed.
Call Stack (most recent call first):
  CMakeLists.txt:26 (find_package)


-- Configuring incomplete, errors occurred!
See also "/build/ktouch-18.12.0/build/CMakeFiles/CMakeOutput.log".
builder for '/nix/store/5rdm9n26gmc1wqqwx3mbqy0awksgp9s1-ktouch-18.12.0.drv' failed with exit code 1
error: build of '/nix/store/5rdm9n26gmc1wqqwx3mbqy0awksgp9s1-ktouch-18.12.0.drv' failed
```
### Steps to reproduce

1. nix-env -i ktouch

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.20, NixOS, 19.03pre168613.ffc604e5579 (Koi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.2`
 - channels(root): `"nixos-19.03pre169895.fa82ebccf66"`
 - channels(bill): `"home-manager"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

--- a/pkgs/applications/kde/ktouch.nix
+++ b/pkgs/applications/kde/ktouch.nix
@@ -3,7 +3,7 @@
 , kconfig, kconfigwidgets, kcoreaddons, kdeclarative, ki18n
 , kitemviews, kcmutils, kio, knewstuff, ktexteditor, kwidgetsaddons
 , kwindowsystem, kxmlgui, qtscript, qtdeclarative, kqtquickcharts
-, qtx11extras, qtgraphicaleffects, xorg
+, qtx11extras, qtgraphicaleffects, qtxmlpatterns, xorg
 }:
 
 
@@ -19,7 +19,8 @@
       kconfig kconfigwidgets kcoreaddons kdeclarative ki18n
       kitemviews kcmutils kio knewstuff ktexteditor kwidgetsaddons
       kwindowsystem kxmlgui qtscript qtdeclarative kqtquickcharts
-      qtx11extras qtgraphicaleffects xorg.libxkbfile xorg.libxcb
+      qtx11extras qtgraphicaleffects qtxmlpatterns
+      xorg.libxkbfile xorg.libxcb
     ];
 
     enableParallelBuilding = true;

